docs/session-inspector/gestures.md

```diff
@@ -10,15 +10,18 @@ tab shows the Saved Gestures screen.
 ## Saved Gestures
 
 The default saved gestures list is empty, but can be populated by creating and saving new gestures.
-This can be done by pressing the `+` button in the bottom left, which will open the
-[Gesture Builder](#gesture-builder) screen.
+New gestures can either be created manually or can be imported from JSON files that are previously downloded from the saved gestures section.
```

Comment:
There is still a typo in `downloded`. I would also use past tense (`that were previously downloaded`)